### PR TITLE
[Instrument Manager] Improving Instrument Manager TestPlan

### PR DESCRIPTION
**Pull request by Jkat**
## Brief summary of changes
fixing some small mistakes in Instrument Manager TestPlan

### CODE REVIEW

File: modules/instrument_manager/test/TestPlan.md [hunk 1]
```diff
@@ -4,10 +4,10 @@
    [Automation Testing]
 2. Check that `Instrument Manager` menu-item exists on a default install under Admin menu-item 
    [Automation Testing]
-3. Check that the warning text is accurate if the `project/instruments` or `project/sql_tables` directories are not writable by Apache. The select file/upload panel should only be there if it's writable.
+3. Check that the warning text is accurate if the `project/instruments` or `project/tables_sql` directories are not writable by Apache. The select file/upload panel should only be there if it's writable.
    [Manual Testing]
 4. Take the sample .linst instrument file `test_all_fields.linst` in `docs/instruments` and upload it in Instrument Manager.
```
Comment (johnsaigle) [inline modules/instrument_manager/test/TestPlan.md]: I think this file doesn't exist anymore actually. I'm not sure where it got moved or if it was just deleted.

Comment (Jkat) [reply modules/instrument_manager/test/TestPlan.md]: Yeah you're right, I already made a note of this on the project card but I just realized right now that an issue should be created! 
https://github.com/aces/Loris/issues/5527

File: modules/instrument_manager/test/TestPlan.md [hunk 2]
```diff
@@ -4,10 +4,10 @@
...
-Check that instrument gets installed properly - instruments/ tables_sql/ . 
+Check that instrument gets installed properly - `instruments/` & `tables_sql/`.
```
Comment (johnsaigle) [inline modules/instrument_manager/test/TestPlan.md]:
```suggestion
Check that instrument gets installed properly. (New files should exists in `project/instruments/` & `project/tables_sql/`.) 
```

**Comment by ridz1208** [comment]
@Jkat is this going to the right branch ? did you mean to issue it on 22.0-release

**Comment by Jkat** [comment]
Ok I will change it

**Pull request merged**